compiler/include/TryStmt.h

```diff
@@ -26,25 +26,31 @@ class TryStmt : public Stmt
 {
 
 public:
+
+  static BlockStmt*   build(bool tryBang, Expr*      expr);
   static BlockStmt*   build(bool tryBang, BlockStmt* body);
+  static BlockStmt*   build(bool tryBang, BlockStmt* body, BlockStmt* catches);
 
-                      TryStmt(bool tryBang, BlockStmt* body);
+                      TryStmt(bool tryBang, BlockStmt* body, BlockStmt* catches);
                       ~TryStmt();
   BlockStmt*          body() const;
   bool                tryBang() const;
 
   void                accept(AstVisitor* visitor);
-  Expr*               copy(SymbolMap* map = NULL, bool internal = false);
-  Expr*               copyInner(SymbolMap* map);
   void                replaceChild(Expr* old_ast, Expr* new_ast);
   Expr*               getFirstChild();
   Expr*               getFirstExpr();
   Expr*               getNextExpr(Expr* expr);
   GenRet              codegen();
+  DECLARE_COPY(TryStmt);
+
+  BlockStmt*          _body;
```

Comment:
I am dealing with the same issue in my ForallStmt, which Michael commented on. To me it is a bummer to let AST_CHILDREN_CALL prevent a field from being private. Given that we are already using accessor methods instead of fields for BlockStmt (see ex. initBlockGet() et al.), I favor doing that for TryStmt as well, keeping its _body private.
Feel free to leave things as-is, if you feel strongly.